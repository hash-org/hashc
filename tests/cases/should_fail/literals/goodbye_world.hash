// run=fail, stage=parse

print("goodbye world...
