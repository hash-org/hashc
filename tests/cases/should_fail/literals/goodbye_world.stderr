error: unclosed string literal
 --> $DIR/goodbye_world.hash:1:8
1 |   print("goodbye world...
  |          ^^^^^^^^^^^^^^^^^ here
